api/capi/include/nnstreamer.h

```diff
@@ -843,6 +843,7 @@ int ml_tensors_info_set_tensor_name (ml_tensors_info_h info, unsigned int index,
  * @brief Gets the tensor name with given handle of tensors information.
  * @since_tizen 5.5
  * @remarks If the function succeeds, @a name should be released using g_free().
+ * @remarks Before 6.0 this function returned the internal pointer so application developers do not need to free. Since 6.0 the name string is internally copied and returned so developers should be released by calling g_free().
```

Comment:
how about removing sentence, 
Since 6.0 the name string is internally copied and returned. So if the function succeeds, @a name should be released using g_free().

Reply:
ACK!